go.mod

```diff
@@ -6,6 +6,7 @@ require (
 	github.com/PagerDuty/go-pagerduty v1.4.1
 	github.com/fsnotify/fsnotify v1.5.1 // indirect
 	github.com/golang/mock v1.6.0
+	github.com/google/go-querystring v1.0.0
```

Comment:
@MitaliBhalla do you need this ? if not please remove

Reply:
Will be needing this as all these are parts of some packages being used, also , I ll be needing gomock soon for mocking the API , this will come in handy when writing tests. Currently, if I remove any of them, it gives me errors